routes/payroll/controllers.js

```diff
@@ -0,0 +1,54 @@
+import PayrollService from "../../services/payroll.js";
+import httpResponse from "../../utils/httpResponse.js";
+
+const controller = {
+    add: async (req, res) => {
+        const addResponse = await PayrollService.add(req.body);
+        if (addResponse.message === "success") {
+            return httpResponse.CREATED(res, addResponse.data);
```

Comment:
Status Code?

Reply:
Sir status code is in httpResponse which contain three variables status , response , message.

Reply:
Sir all the status code are in the httpResponse i am just calling it and using it.